k8s/upgrade/src/helm/chart.rs

```diff
@@ -128,178 +128,178 @@ impl HelmValuesCollection for CoreValues {
-    pub(crate) fn grafana_download_dashboards_image_tag(&self) -> &str {
+    pub fn grafana_download_dashboards_image_tag(&self) -> &str {
         self.loki_stack.grafana_download_dashboards_image_tag()
     }
 
     /// This returns the image tag for the grafana/grafana container.
-    pub(crate) fn grafana_image_tag(&self) -> &str {
+    pub fn grafana_image_tag(&self) -> &str {
         self.loki_stack.grafana_image_tag()
     }
 
     /// This returns the image tag for the kiwigrid/k8s-sidecar container.
-    pub(crate) fn grafana_sidecar_image_tag(&self) -> &str {
+    pub fn grafana_sidecar_image_tag(&self) -> &str {
         self.loki_stack.grafana_sidecar_image_tag()
     }
 
     /// This is a getter for the localpv-provisioner sub-chart's release version.
-    pub(crate) fn localpv_release_version(&self) -> &str {
+    pub fn localpv_release_version(&self) -> &str {
         self.localpv_provisioner.release_version()
     }
 
     /// This is a getter for the container image tag of the hostpath localpv provisioner.
-    pub(crate) fn localpv_provisioner_image_tag(&self) -> &str {
+    pub fn localpv_provisioner_image_tag(&self) -> &str {
         self.localpv_provisioner.provisioner_image_tag()
     }
 
     /// This is a getter for the image tag of the localpv helper container.
-    pub(crate) fn localpv_helper_image_tag(&self) -> &str {
+    pub fn localpv_helper_image_tag(&self) -> &str {
         self.localpv_provisioner.helper_image_tag()
     }
 
     /// This is a getter for the prometheus/alertmanager container's image tag.
-    pub(crate) fn prometheus_alertmanager_image_tag(&self) -> &str {
+    pub fn prometheus_alertmanager_image_tag(&self) -> &str {
         self.loki_stack.prometheus_alertmanager_image_tag()
     }
 
     /// This is a getter for the prometheus/node-exporter container's image tag.
-    pub(crate) fn prometheus_node_exporter_image_tag(&self) -> &str {
+    pub fn prometheus_node_exporter_image_tag(&self) -> &str {
         self.loki_stack.prometheus_node_exporter_image_tag()
     }
 
     /// This is a getter for the prom/pushgateway container's image tag.
-    pub(crate) fn prometheus_pushgateway_image_tag(&self) -> &str {
+    pub fn prometheus_pushgateway_image_tag(&self) -> &str {
         self.loki_stack.prometheus_pushgateway_image_tag()
     }
 
     /// This is a getter for the prometheus/prometheus container's image tag.
-    pub(crate) fn prometheus_server_image_tag(&self) -> &str {
+    pub fn prometheus_server_image_tag(&self) -> &str {
         self.loki_stack.prometheus_server_image_tag()
     }
 
     /// This is the value of the deprecated key for log silence configuration.
-    pub(crate) fn deprecated_log_silence_level(&self) -> &str {
+    pub fn deprecated_log_silence_level(&self) -> &str {
         self.base.deprecated_log_silence_level()
     }
 
-    pub(crate) fn jaeger_operator_image_tag(&self) -> &str {
+    pub fn jaeger_operator_image_tag(&self) -> &str {
```

Comment:
missing doc comment

Reply:
I'll add in all of these missing doc comments.